internal/resource-request-operator/broadcaster.go

```diff
@@ -268,3 +268,12 @@ func extractPodResources(pod *corev1.Pod) corev1.ResourceList {
 	resourcesToExtract, _ := resourcehelper.PodRequestsAndLimits(pod)
 	return resourcesToExtract
 }
+
+func isNodeRunning(node *corev1.Node) bool {
```

Comment:
```suggestion
func isNodeReady(node *corev1.Node) bool {
```

Since this function may be of use also in other situations (e.g. VK node controller), I would propose to move this outside of this specific package